### PR TITLE
Initial changes for Narrator known issue

### PR DESCRIPTION
**Pull request by merriemcgaw**
I finally got around to writing this Known Issue found by a bug we fixed for vNext.

### CODE REVIEW

File: releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true
+
+## Symptoms
+Narrator does not announce the value of the ReadOnly property of the System.Windows.Forms.DataGridViewTextBoxColumn. This is the case regardless of the value of the ReadOnly property. This could cause confusion for Narrator users when editing content in a DataGridViewTextBoxColumn.
+## Cause
+The issue is caused by changes to updates in .NET Framework 4.7.2 which improved the accessibility support of the DataGridView. That change caused Narrator to be unable to narrate the ReadOnly status of the TextBoxColumn regardless of the setting.
```
Comment (vivmishra) [inline releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: Is something missing here?

Comment (merriemcgaw) [reply releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: D'oh!  removing "to updates"

File: releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Workaround 
+There is no workaround for this issue.
+
+## Resolution
+A proper fix of the DataGridView AccessibleObject will ship in the coming versions of .NET Framework. This known issue will be updated with more details when that happens.
```
Comment (rpetrusha) [inline releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: nit: in the coming versions --> in coming versions

File: releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md (hunk 1 above)
Comment (rpetrusha) [inline releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: nit: which improved --> that improved
nit: caused Narrator to be unable --> left Narrator unable

File: releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md (hunk 2 above)
Comment (merriemcgaw) [reply releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: Updated

File: releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md (hunk 1 above)
Comment (merriemcgaw) [reply releases/net472/KnownIssues/599936 - Narrator does not annouce the ReadOnly Status for DataGridView TextBox column with the TextBox column Readonly property set as true.md]: Updated

**Comment by rpetrusha** [comment]
@merriemcgaw, you mentioned that you'd updated your PR, but no additional commit appears here.

**Comment by merriemcgaw** [comment]
@rpetrusha , there's another commit from yesterday that has the changes. I can see that there are 3 commits total for this PR. Can you check to see if you can see them?

**Comment by rpetrusha** [comment]
Yes, I can see them, @merriemcgaw. This looks good. You can merge when you're ready.

**Pull request merged**